File: python/7 kyu/375.py
# https://www.codewars.com/kata/5aee96e22c5061ee90000024/train/python

# Passed

def quadratic_gen(a, b, c, start=0, step=1):
    while True:
        result = [start, a * start ** 2 + b * start + c]
        start += step
        yield result

output = next(quadratic_gen(1, 0, 0))
print(output)

File: nodejs/8 kyu/375.js
// https://www.codewars.com/kata/545991b4cbae2a5fda000158/train/javascript

// Passed

function include(arr, item) {
  let result = arr.includes(item);
  return result;
}

const output = include([1, 2, 3, 4], 3);
console.log(output);